pkg/deviceutils/device-utils.go

```diff
@@ -39,6 +39,7 @@ const (
 	diskSDPattern        = "/dev/sd*"
 	diskNvmePath         = "/dev/nvme"
 	diskNvmePattern      = "^/dev/nvme[0-9]+n[0-9]+$"
+	devNvmePattern       = "/dev/nvme*"
```

Comment:
Can we use a different name to make it clear that it's a glob and not a regexp? eg devNvmeGlob